spec/models/application_spec.rb

```diff
@@ -245,6 +247,20 @@
     end
   end
 
+  describe '#age' do
+    subject { application.applicant_age }
+
+    context 'when applicant is born on Feb 29th in a leap year' do
```

Comment:
Is one single scenario enough for all cases? That's just a question, I'm not sure.